package.json

```diff
@@ -1,142 +1,46 @@
+    "format": "prettier --cache --write '**/*.{js,md,mdx,ts,tsx,yml}'",
+    "format:diff": "prettier --cache --list-different '**/*.{js.md,mdx,ts,tsx,yml}'",
     "install:docs": "(cd docs && npm install --legacy-peer-deps)",
     "lint": "eslint '**/*.{js,ts,tsx,md,mdx}' --max-warnings=0",
     "lint:fix": "npm run lint -- --fix",
-    "lint:md": "markdownlint-cli2 \"**/*.{md,mdx}\" \"!.github\" \"!.changeset\" \"!node_modules\" \"!CHANGELOG.md\"",
+    "lint:md": "markdownlint-cli2 \"**/*.{md,mdx}\" \"!.github\" \"!.changeset\" \"!**/node_modules/**\" \"!**/CHANGELOG.md\"",
+    "prepare": "husky install",
+    "release": "npm run build && changeset publish",
+    "setup": "./script/setup",
+    "size": "size-limit",
+    "start": "concurrently npm:start:*",
+    "start:docs": "cd docs && npm run develop",
+    "start:storybook": "STORYBOOK=true start-storybook -p 6006",
     "test": "jest",
-    "test:storybook": "test-storybook",
+    "test:coverage": "npm run test -- --coverage",
     "test:update": "npm run test -- --updateSnapshot",
     "test:watch": "npm run test -- --watch",
-    "test:coverage": "npm run test -- --coverage",
-    "test:type-check": "tsc --noEmit",
-    "release": "npm run build && changeset publish",
-    "size": "size-limit",
-    "prepare": "husky install"
-  },
-  "repository": "primer/react",
-  "keywords": [
-    "react",
-    "components",
-    "library",
-    "design-system"
-  ],
-  "files": [
-    "codemods",
-    "dist",
-    "lib",
-    "lib/node_modules",
-    "lib-esm",
-    "css",
-    "index.d.ts",
-    "deprecated/package.json",
-    "drafts/package.json",
-    "!lib/__tests__",
-    "!lib/stories",
-    "!lib-esm/__tests__",
-    "!lib-esm/stories",
-    "generated"
-  ],
-  "author": "GitHub, Inc.",
-  "license": "MIT",
-  "engines": {
-    "node": ">=12",
-    "npm": ">=7"
-  },
-  "dependencies": {
-    "@github/combobox-nav": "^2.1.5",
-    "@github/markdown-toolbar-element": "^2.1.0",
-    "@github/paste-markdown": "^1.4.0",
-    "@github/relative-time-element": "^4.1.2",
-    "@lit-labs/react": "1.1.1",
-    "@primer/behaviors": "1.3.4",
-    "@primer/octicons-react": "^19.1.0",
-    "@primer/primitives": "7.11.10",
-    "@react-aria/ssr": "^3.1.0",
-    "@styled-system/css": "^5.1.5",
-    "@styled-system/props": "^5.1.5",
-    "@styled-system/theme-get": "^5.1.2",
-    "@types/styled-components": "^5.1.11",
-    "@types/styled-system": "^5.1.12",
-    "@types/styled-system__css": "^5.0.16",
-    "@types/styled-system__theme-get": "^5.0.1",
-    "classnames": "^2.3.1",
-    "color2k": "^2.0.0",
-    "deepmerge": "^4.2.2",
-    "focus-visible": "^5.2.0",
-    "fzy.js": "0.4.1",
-    "history": "^5.0.0",
-    "react-intersection-observer": "9.4.3",
-    "styled-system": "^5.1.5"
+    "test:type-check": "tsc --noEmit"
   },
   "devDependencies": {
-    "@actions/core": "1.10.0",
-    "@babel/cli": "7.21.0",
-    "@babel/core": "7.21.0",
+    "@babel/core": "7.21.8",
```

Comment:
Do we keep the dependencies that are going to be used across packages on the root package.json?

Reply:
@broccolinisoup that would definitely be the rule of thumb I'd love to follow. Basically any workspace should list the dependencies that it needs in order to run function. That can be in a child workspace like `@primer/react` or can be at the top-level if we have scripts there that depend on those dependencies.

Reply:
Oh right right! Make sense.